src/parser/monk/mistweaver/modules/core/Channeling.tsx

```diff
@@ -22,7 +24,7 @@ class Channeling extends CoreChanneling {
     super.on_byPlayer_cast(event);
   }
 
-  cancelChannel(event, ability) {
+  cancelChannel(event: CastEvent, ability: {type: EventType; timestamp: number; ability: Ability; sourceID: number; isCancelled: boolean}) {
```

Comment:
the ability part of this seems odd, wouldn't that be `ability: Ability`?

Reply:
You're right. I was miss reading what was passed in from the parent